crypto/keys/multisig/multisig.go

```diff
@@ -25,9 +25,6 @@ func NewLegacyAminoPubKey(threshold int, pubKeys []cryptotypes.PubKey) *LegacyAm
 	if threshold <= 0 {
 		panic("threshold k of n multisignature: k <= 0")
 	}
-	if len(pubKeys) < threshold {
```

Comment:
that's a commit from another PR right? I think it should be reverted from this PR